.circleci/config.yml

```diff
@@ -140,14 +140,18 @@ build_and_deploy: &build_and_deploy
             # Example: Merge pull request #35 from eGovPDX/powr-152
             cd /var/www/html
             # Must read the last commit message from Github. The last one in local repo is "CircleCI Deployment"
-            if [[ $(git log origin/master --pretty=format:%s -1) =~ (^Merge pull request #[0-9]* from eGovPDX\/)([^,]*) ]]; then
-              echo "Found a Pull Request Merge for branch: '${BASH_REMATCH[2]}'";
+            last_log=$(git log origin/master --pretty=format:%s -1)
+            lowercase_last_log=`php -r "print strtolower('$last_log');"`
+            if [[ $lowercase_last_log =~ ^(powr-[0-9]+)[[:space:]]+.+$ ]]; then
+              echo "Found the branch name: '${BASH_REMATCH[1]}'";
               # Check if a multidev environment with the branch name exists or not
               terminus multidev:list --format=list --field=Name portlandor > ./env_list.txt
-              if grep -Fxq "${BASH_REMATCH[2]}" ./env_list.txt; then
-                # Delete both the multidev site and branch
-                terminus multidev:delete --delete-branch portlandor.${BASH_REMATCH[2]} -y
-              fi
+              # Pattern to match "powr-1234a" or "powr-1234-1". Not "powr-12345"
+              pattern="^${BASH_REMATCH[1]}([a-z]*|(-\d+))\s$" && echo $pattern
```

Comment:
I think we should also match the pattern powr-1234-a where 'a' can be 1 or more letters.

Reply:
Great feedback! Pattern updated accordingly.